do-unikraft-bincompat

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
```

Comment:
If you run the following sequence of commands:

```bash
./do-unikraft-bincompat setup
./do-unikraft-bincompat build
./do-unikraft-bincompat run_build
./do-unikraft-bincompat run_build_debug
```

there is not the expected result. I think we may print some message as a warning when given the `run_build_debug` paramter.